starlite/app.py

```diff
@@ -135,7 +135,7 @@ def __init__(
         after_response: Optional["AfterResponseHookHandler"] = None,
         after_shutdown: Optional["SingleOrList[LifeSpanHookHandler]"] = None,
         after_startup: Optional["SingleOrList[LifeSpanHookHandler]"] = None,
-        allowed_hosts: Optional[List[str]] = None,
+        allowed_hosts: Optional[Union[List[str], "AllowedHostsConfig"]] = None,
```

Comment:
Since allowed_hosts can accept `AllowedHostsConig` should we mention this in the docs https://starlite-api.github.io/starlite/usage/7-middleware/3-builtin-middlewares/2-allowed-hosts-middleware/ ?

Reply:
added.